pkg/api/plugin/api/config.go

```diff
@@ -17,6 +18,8 @@ type Config struct {
 	// SSH to system nodes allowed IP ranges
 	SSHSourceAddressPrefixes []string `json:"sshSourceAddressPrefixes,omitempty"`
 
+	AzureFileStorageAccount string `json:"azureFileStorageAccount,omitempty"`
```

Comment:
should not be included in pkg/api/plugin, this is not a configurable that the RP sends to the plugin